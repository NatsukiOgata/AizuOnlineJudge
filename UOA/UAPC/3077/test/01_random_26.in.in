348607469187475 9 84
