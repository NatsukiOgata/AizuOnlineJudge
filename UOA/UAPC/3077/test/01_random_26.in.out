3137467222687275000000000000000000000000000000000000000000000000000000000000000000000000000000000000
